Mathlib/Algebra/Ring/BooleanRing.lean

```diff
@@ -363,7 +363,7 @@ instance [Inhabited α] : Inhabited (AsBoolRing α) :=
   ‹Inhabited α›
 
 -- See note [reducible non-instances]
-/-- Every generalized Boolean algebra has the structure of a non unital commutative ring with the
+/-- Every generalized Boolean algebra has the structure of a nonunital commutative ring with the
```

Comment:
I'd use a hyphen here: that seems more common, given my anecdotal math evidence.

Reply:
Changed to non-unital, since either option is pretty rare in literature.